### ISSUE TITLE
Rearrange the class names, file names

### ISSUE THREAD

**Issue by noirgif** [bigeagle/danmaQ#47]
Since the class name of main window is changed, the file names and class names doesn't match quite well. So it's preferable to rename these files, for example, like below:

```
danmaku.h -> danmaq.hpp
danmaku_ui.{cpp, h} -> Danmaku.{cpp, hpp}
danmaku_window.{cpp, h} -> DMWindow.{cpp, hpp}
danmaq_app.{cpp, h} -> DMMainWindow.{cpp, hpp}
```

The include directives, the translation file, and cmake file are subject to this change.

Also, I'd recommend some class name change, but it's just my personal taste and I'll leave it open to comment :)
* 'Comment' is better since 'danmaku' refers to a barrage of comments, not a single one.
* We can replace 'DMWindow' with 'DMCanvas' or something else, since it's not really a window, but I'm not sure the word 'canvas' is good enough.

**Comment by Harry-Chen** [comment]
Actually I am planning on this and I quite agree with you.
Since I am busy preparing for exams these days, it would be nice if you could offer help on this.
In terms of details, 'DMCanvas' is a good name. Also I prefer `common.hpp` instead of `danmaq.hpp`. Other suggestions are good, just do it!

**Comment by Harry-Chen** [comment]
Solved by 872a6a0fdf9e916303213c215a981a1320627dd8.
